core-web/libs/portlets/edit-ema/portlet/src/lib/store/models.ts

```diff
@@ -0,0 +1,42 @@
+import { CurrentUser } from '@dotcms/dotcms-js';
+import { DotExperiment, DotLanguage, DotPageToolUrlParams } from '@dotcms/dotcms-models';
+import { InfoPage } from '@dotcms/ui';
+
+import { DotPageApiParams, DotPageApiResponse } from '../services/dot-page-api.service';
+import { UVE_STATUS } from '../shared/enums';
+import { DotDeviceWithIcon, DotPage, NavigationBarItem } from '../shared/models';
+
+export interface UVEState {
+    $isEnterprise: boolean;
+    $pageAPIResponse?: DotPageApiResponse;
+    $languages: DotLanguage[];
+    $currentUser?: CurrentUser;
+    $experiment?: DotExperiment;
+    $error?: number;
+    $params?: DotPageApiParams;
+    $status: UVE_STATUS;
+    $isTraditionalPage: boolean;
+    $canEditPage: boolean;
+    $pageIsLocked: boolean;
```

Comment:
I think I said this... but I am not sure we need the `$` in the interface but only in the properties in the components.

Reply:
Here in the model, they look like properties, but when we access them from the store they are signals, so I think is fine to leave them like this

Reply:
Yes, but when you access this from the component class you should to like

```
class ComponentClass {
    $theStoreProp = store.someProp
}
```

And that is how we identify `$theStoreProp` as a signal, because in SignalStore we know every prop in the store is a Signal by default.

Reply:
That makes sense to me. Will change